rclcpp/src/rclcpp/executor.cpp

```diff
@@ -60,7 +60,8 @@ Executor::Executor(const ExecutorArgs & args)
   // Store the context for later use.
   context_ = args.context;
 
-  ret = rcl_wait_set_init(&wait_set_, 0, 2, 0, 0, 0, context_->get_rcl_context().get(), allocator);
+  ret = rcl_wait_set_init(&wait_set_, 0, 2, 0, 0, 0, 0,
+      context_->get_rcl_context().get(), allocator);
```

Comment:
This should also be two spaces.

Reply:
Turns out this is also caused by `ament_uncrustify`. Should I still fix this indentation?

Reply:
Nah, life's too short :D

If it bothers you, you can try to figure out what the issue is. We have the ability to patch uncrustify as we distribute it ourselves.

Reply:
You can also try to reform the line, if you wish.

```
ret = rcl_wait_set_init(
  &wait_set_,
  0, 2, 0, ...,
  context_->get_rcl_context().get(),
  allocator);
```

Or something like that.

Reply:
I have manually removed the extra spaces in the indentation for now.